From a8ea135125b284a8a9c965d8091b3d1d9a31849c Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Tue, 5 Apr 2022 16:51:34 +0200
Subject: [PATCH] CMakeLists.txt: add missing endian.h check

This is needed to actually have HAVE_ENDIAN_H defined properly in config.h.

Upstream-Status: Submitted [https://gitlab.freedesktop.org/mesa/piglit/-/merge_requests/648]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9fb0508da..158fac4bc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -481,6 +481,7 @@ check_include_file(sys/stat.h  HAVE_SYS_STAT_H)
 check_include_file(unistd.h    HAVE_UNISTD_H)
 check_include_file(fcntl.h     HAVE_FCNTL_H)
 check_include_file(linux/sync_file.h HAVE_LINUX_SYNC_FILE_H)
+check_include_file(endian.h    HAVE_ENDIAN_H)
 
 if(DEFINED PIGLIT_INSTALL_VERSION)
 	set(PIGLIT_INSTALL_VERSION_SUFFIX
